### ISSUE TITLE
Definition for rule 'no-commonjs' was not found

### ISSUE THREAD

**Issue by Telokis** [benmosher/eslint-plugin-import#853]
I get this error in every file of my project.

`1:1  error  Definition for rule 'no-commonjs' was not found  no-commonjs`

Here is my config:

```js
{
    "extends": [
        "plugin:react/recommended",
        "airbnb-base"
    ],
    "parserOptions": {
        "ecmaVersion": 2017,
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "plugins": [
        "react",
        "import"
    ],
    "rules": {
        // Overriden
        "no-underscore-dangle": "off",
        "func-names": ["error", "as-needed"],
        "no-plusplus": "off",
        "no-commonjs": "error",
        "indent": ["error", 4],
        "no-var": "error",
        "class-methods-use-this": "off",
        "no-param-reassign": ["error", { "props": false }],
        "no-useless-return": "warn"
    }
}```

I am not doing anything particular. I have no idea at all about this issue.

**Comment by ljharb** [comment]
What does `npm ls` report?
